doc/features.md

```diff
@@ -127,3 +127,22 @@ Beware that `CONFIG` section contents must comply with YAML syntax restrictions:
 * `- "silex/silex: *"` without quotes is an invalid YAML.
 * `- "silex/silex: ~1.2"` without quotes is a YAML object and refused by melody.
 * `- "-S"` without quotes is an array of arrays.
+
+
+Using fork and private repositories
+-----------------------------------
+
+If you need to use packages not registred in Packagist repository, you can specify repositories
```

Comment:
could you please wrap your lines after the first word that crosses the 72th character?